tox.ini

```diff
@@ -1,32 +1,32 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,py36,pypy,pypy3,coverage,docs
+    py27,py34,py35,py36,pypy,pypy3
 
 [testenv]
+usedevelop = true
 commands =
     coverage run -m zope.testrunner --test-path=src []
+    coverage run -a -m sphinx -b doctest -d {envdir}/.cache/doctrees docs {envdir}/.cache/doctest
 deps =
-    .[test]
+    .[test,docs]
     coverage
 setenv =
    COVERAGE_FILE=.coverage.{envname}
 
 [testenv:coverage]
+usedevelop = true
 setenv =
   COVERAGE_FILE=.coverage
-skip_install = true
 commands =
     coverage erase
     coverage combine
-    coverage report
-    coverage html
-    coverage xml
+    coverage report -i
+    coverage html -i
+    coverage xml -i
```

Comment:
`skip_install` seems correct to me:  `coverage` doesn't actually import the code, AFAIK.